Makefile.Common

```diff
@@ -4,7 +4,7 @@ ALL_PKGS := $(sort $(shell go list ./...))
 # COVER_PKGS is the list of packages to include in the coverage
 COVER_PKGS := $(shell go list ./... | tr "\n" ",")
 
-GOTEST_TIMEOUT?=120s
+GOTEST_TIMEOUT?=240s
```

Comment:
If we're seeing timeouts in all our tests, we can probably remove the manual increase in the Windows unit tests workflow, which I just bumped to 240s as well.

Reply:
Yeah I was testing if the timeout would help (it did), so I made your change from #9992 apply to every test suite

Reply:
It's pretty small, but it would be better if this change was not in this PR :)